frontend/src/components/WebBrowser/websites/Bluestar/index.js

```diff
@@ -92,6 +87,8 @@ class MyBluestar extends React.Component {
 
   render() {
     const { logBrowserActions, actionTypes } = this.props;
+    const accounts = getAllAccounts();
```

Comment:
Can use getAccount here:

```suggestion
    const account = getAccount('fa59b235-284f-40ba-aae7-dtf48689d22');
```